enterprise/wsproxy/wsproxy.go

```diff
@@ -60,7 +59,8 @@ type Options struct {
 	SecureAuthCookie bool
 	DisablePathApps  bool
 
-	ProxySessionToken string
+	ProxySessionToken     string
+	DangerousAllowAllCors bool
```

Comment:
I don't think we should prefix struct properties with their severity. Ideally the naming of the variable indicates how damaging it is to specify an option.

Reply:
For the flag it's different because it's grouped in dangerous.